common/djangoapps/student/tests/test_activate_account.py

```diff
@@ -158,7 +158,9 @@ def test_account_activation_message_disabled(self):
         self.login()
         expected_message = render_to_string(
             'registration/activate_account_notice.html',
-            {'email': self.user.email}
+            {'email': self.user.email
+
+        }
```

Comment:
Did something unintended happen here? Looks like the formatting might have been a mistake.